docs: Search Metadata Batch 03 (#14593)
hashicorp/consul: website/content/docs/connect/registration/sidecar-service.mdx

---
layout: docs

> page_title: Service Mesh Proxy: Sidecar Configuration

Reviewer: I don't like how clunky the title I suggested is, but it's a more accurate reflection of what is going on (and how it's different from the alternative).
```suggestion
page_title: Service Mesh Proxy Registration: Included in Service Registration
```

description: >-

> In a service mesh, sidecar proxies run in the same node as service instances and are configured in service definition files. Learn about default settings, customizable parameters, limitations, and lifecycle behaviors.

Reviewer: ```suggestion
You can conveniently register a service instance and its sidecar proxy together. Learn about default settings, customizable parameters, limitations, and lifecycle behaviors of the sidecar proxy.
```

---

> # Service Mesh Proxy Sidecar Configuration

Reviewer: Same thoughts as the `page_title` comment above
